src/main/java/com/epam/lowcost/util/EndPoints.java

```diff
@@ -19,6 +19,7 @@
     public static final String NEW_TICKET = "/new-ticket";
     public static final String RETURN = "/return";
     public static final String FLIGHT = "/flight";
+    public static final String SET_FLIGHT_BY_PAGE = "/setFlightsByPage";
```

Comment:
why camelCase ???????
+ u can use only `/page`

Reply:
Changed all /entity-by-page to simply /page